src/web/app/components/question-types/question-statistics/question-statistics-calculation/constsum-recipients-question-statistics-calculation.ts

```diff
@@ -55,6 +57,19 @@ export class ConstsumRecipientsQuestionStatisticsCalculation
       this.totalPointsPerOption[recipient] = sum;
       this.averagePointsPerOption[recipient] = +(answers.length === 0 ? 0 : sum / answers.length).toFixed(2);
     }
+
+    let totalSum: number = 0;
+    let totalStudents: number = 0;
+    for (const recipient of Object.keys(this.pointsPerOption)) {
+      totalSum += this.totalPointsPerOption[recipient];
+      totalStudents += 1;
+    }
```

Comment:
It looks like `totalSum` is the same as `sum` as defined previously, and `totalStudents` can be found easily through `answers.length` too (similar to how the average points per option was calculated above)

Reply:
thanks for helping look through the code!

it would seem that the `sum` might have caused some misunderstanding here, as `sum` refers to the sum of a student's points received (which will be reflected under the **Total Points** column)

`totalSum` on the other hand calculates the sum of all the student's total scores (in the above screenshot example, that would be 800)

on the same note, `answers.length` always returns 1 for each student as each of them has only one instance of receiving points in the above example

however I did notice that this extra loop was unnecessary and could be omitted, so thanks so much for pointing this out!

Reply:
You're right, I see the intent behind introducing `totalSum` and `totalStudents` now. However, why should we need to find the total sum of scores across students, or the total number of students, at all?